its/pom.xml

```diff
@@ -12,20 +12,14 @@
   <name>SonarSource XML Analyzer :: ITs</name>
   <packaging>pom</packaging>
 
-  <dependencyManagement>
-    <dependencies>
-      <dependency>
-        <groupId>junit</groupId>
-        <artifactId>junit</artifactId>
-        <version>4.13.2</version>
-        <scope>test</scope>
-      </dependency>
-    </dependencies>
-  </dependencyManagement>
-
   <modules>
     <module>plugin</module>
     <module>ruling</module>
   </modules>
 
+  <properties>
+    <skipTests>true</skipTests>
+    <maven.deploy.skip>true</maven.deploy.skip>
+  </properties>
```

Comment:
What changed in our configuration that is likely to deploy our ITs to repox or elsewhere?

Reply:
As discussed, now that we added the `its` module as a submodule of the root, we should explicitly exclude from deployment since it is not meant to be used in production.